src/helpers/index.ts

```diff
@@ -1,9 +1,6 @@
 import logMetrics from './log-metrics';
 import axiosErrorInterceptor from './interceptors/axios-error-interceptor';
 import getLock from './get-lock';
+import getRootSpan from './get-root-span';
 
-export {
-  logMetrics,
-  axiosErrorInterceptor,
-  getLock
-};
+export { logMetrics, axiosErrorInterceptor, getLock, getRootSpan };
```

Comment:
Nice! Shall we document getRootSpan in our public docs? Or do we prefer this being kept "internal" ?

Reply:
I believe we can keep it internal for now, until either we use it in all our apps and everything works as expected or it is documented in dd-trace lib. Wdyt?